src/components/axes/timeAxis.ts

```diff
@@ -240,8 +242,14 @@ export module Axis {
 
     public _computeHeight() {
       var textHeight = this._measurer.measure().height;
-      this._tierHeights = this._tierLabelPositions.map((pos: string) =>
-        textHeight + this.tickLabelPadding() + ((pos === "between") ? 0 : this._maxLabelTickLength()));
+      var maximumTiers = _Util.Methods.max(this._possibleTimeAxisConfigurations.map((config: TimeAxisConfiguration) => config.length), 0);
```

Comment:
For completeness, I am adding the offline conversation. Yes, we will have extra space, but if we zoom in and out, at some point we will hit the 2 tier configuration and we don't want our graph to jump at this point